pyvelox/pyvelox.h

```diff
@@ -419,9 +360,26 @@ inline void addVectorBindings(
           })
       .def("encoding", &BaseVector::encoding)
       .def("append", [](VectorPtr& u, VectorPtr& v) { appendVectors(u, v); });
+
+  for (int8_t t = 0; t <= static_cast<int8_t>(TypeKind::INTERVAL_DAY_TIME);
+       t++) {
+    // VARCHAR and VARBINARY create the same C++ type, so we skip one of them.
+    if (static_cast<TypeKind>(t) != TypeKind::VARCHAR) {
+      VELOX_DYNAMIC_SCALAR_TYPE_DISPATCH(
+          registerTypedVectors,
+          static_cast<TypeKind>(t),
+          m,
+          asModuleLocalDefinitions);
+    }
+  }
+
   m.def("from_list", [](const py::list& list) mutable {
     return pyListToVector(list, PyVeloxContext::getInstance().pool());
   });
+  m.def("constant_vector", [](const py::handle& obj, vector_size_t length) {
```

Comment:
I think we can extend this and add the type as an optional named parameter ?
If the type is supplied we can use that to ensure that variants are all of the particular type else type can be determined from varints. This will also allow us to create vectors which are all null etc.

Reply:
Added type as optional third parameter